test/handlers/CompatibilityFallbackHandler.spec.ts

```diff
@@ -92,6 +93,24 @@ describe("CompatibilityFallbackHandler", async () => {
             const sig2 = await signHash(user2, calculateSafeMessageHash(validator, "0xbaddad", await chainId()))
             expect(await validator.callStatic['isValidSignature(bytes,bytes)']("0xbaddad", buildSignatureBytes([sig1, sig2]))).to.be.eq("0x20c13b0b")
         })
+
+        it('should pass a keccak256 hash of the pre-image message to support contract signatures', async () => {
```

Comment:
i'm a little bit unsure if the test description is the best one, please let me know if you have better ideas

Reply:
How is this test different from the one above? That is something I would love to understand from the name of this test.

As far as I can tell you use create an additional Safe and use it as the owner of the other Safe, so you test nested Safes with contract signatures. I would mention this in the title.

So this is already partially mentioned with "support contract signatures" ... I would remove the part about keccak and pre-image.

Reply:
I haven't spotted the one above. I will remove this test and extend the other one by adding a smart contract wallet as an owner.